Right fix in another shape: Django's PermissionRequiredMixin, after
WgerFormMixin, enforces the permission_required the view already names.

Fixes: s2-admin-config-no-permission

diff --git a/wger/gym/views/admin_config.py b/wger/gym/views/admin_config.py
index 80846ed..8209291 100644
--- a/wger/gym/views/admin_config.py
+++ b/wger/gym/views/admin_config.py
@@ -18,6 +18,7 @@
 import logging
 
 # Django
+from django.contrib.auth.mixins import PermissionRequiredMixin
 from django.urls import reverse
 from django.utils.translation import (
     gettext as _,
@@ -33,7 +34,7 @@ from wger.utils.generic_views import WgerFormMixin
 logger = logging.getLogger(__name__)
 
 
-class ConfigUpdateView(WgerFormMixin, UpdateView):
+class ConfigUpdateView(WgerFormMixin, PermissionRequiredMixin, UpdateView):
     """
     View to update an existing admin gym configuration
     """
